### ISSUE TITLE
Results and Pre-Trained Weights

### ISSUE THREAD

**Issue by ed-fish** [42Shawn/LCR_BNN#2]
Hi, I have trained the model as described in the README but do not obtain the same results reported in the paper. 

Below are the results I obtained with `paper' and `code' being the results reported and the results from the repo respectively. 

<html xmlns:v="urn:schemas-microsoft-com:vml"
xmlns:o="urn:schemas-microsoft-com:office:office"
xmlns:x="urn:schemas-microsoft-com:office:excel"
xmlns="http://www.w3.org/TR/REC-html40">

<head>

<meta name=ProgId content=Excel.Sheet>
<meta name=Generator content="Microsoft Excel 15">
<link id=Main-File rel=Main-File
href="file:///C:/Users/EDWARD~1.FIS/AppData/Local/Temp/msohtmlclip1/01/clip.htm">
<link rel=File-List
href="file:///C:/Users/EDWARD~1.FIS/AppData/Local/Temp/msohtmlclip1/01/clip_filelist.xml">
<style>

</style>
</head>

<body link="#0563C1" vlink="#954F72">

Dataset | Model | Method | BW | Top 1 | Top 5
-- | -- | -- | -- | -- | --
ImageNet | ResNet18 | IR-Net | 1-1 | 58.1 | 80
ImageNet | ResNet18 | LCR_BNN Paper | 1-1 | 59.6 | 81.6
ImageNet | ResNet18 | LCR_BNN Code | 1-1 | 57.8 | 80.2
  |   |   |   |   |  
CIFAR 10 | ResNet18 | IR-Net | 1-1 | 91.5 | NA
CIFAR 10 | ResNet18 | LCR_BNN Paper | 1-1 | 91.8 | NA
CIFAR10 | ResNet18 | LCR_BNN Code | 1-1 | 87.6 | 99.5

</body>

</html>

Please can you provide pre-trained model weights or further training instructions. Both models were trained for 150 Epochs using the same learning rate and scheduling as described in the paper. Thank you.
